src/controller/LottoGameController.js

```diff
@@ -0,0 +1,89 @@
+const Calculator = require("../model/Calculator");
+const { MESSAGE_ACCORDING_ASK } = require("../constants/Message");
+const MyNumberGenerator = require("../model/Generator");
+const Lotto = require("../model/Lotto");
+
+const { readLine, close } = require("../utils/MissionUtils");
+const View = require("../view/View");
+
+class LottoGameController {
+  #winningCount;
+  #winningNumbers;
+  #moneyInput;
+  #model = {
+    generatorModel: new MyNumberGenerator(),
+    calculatorModel: new Calculator(),
+  };
+  #view = { print: new View() };
+
+  start() {
+    this.#inputMoneyFromUser();
+  }
+
+  #inputMoneyFromUser() {
+    readLine(MESSAGE_ACCORDING_ASK.INPUT_MONEY, (moneyInput) => {
+      this.#moneyInput = moneyInput;
+      this.#generateMyNumbers();
+    });
+  }
+
+  #generateMyNumbers() {
+    const myLottoNumbers = this.#model.generatorModel.generateMyLottoNumber(
+      this.#moneyInput
+    );
+    this.#notifyMyInfo(myLottoNumbers);
+    this.#getWinningNumbersFromUser(myLottoNumbers);
+  }
+
+  #notifyMyInfo(myLottoNumbers) {
+    this.#notifyMyLottoSize(myLottoNumbers);
+    this.#notifyMyLottoList(myLottoNumbers);
+  }
+
+  #notifyMyLottoSize(myLottoNumbers) {
+    this.#view.print.purchasedSize(myLottoNumbers);
+  }
+
+  #notifyMyLottoList(myLottoNumbers) {
+    this.#view.print.purchasedList(myLottoNumbers);
+  }
+
+  #getWinningNumbersFromUser(myLottoNumbers) {
+    readLine(MESSAGE_ACCORDING_ASK.INPUT_WINNING_NUMBER, (winningNumber) => {
+      this.#getBonusNumberFromUser(myLottoNumbers, winningNumber);
+    });
+  }
+
+  #getBonusNumberFromUser(myLottoNumbers, winningNumber) {
+    readLine(MESSAGE_ACCORDING_ASK.INPUT_BONUS_NUMBER, (bonusNumber) => {
+      const lottoModel = new Lotto({
+        winningNumber,
+        bonusNumber: +bonusNumber,
```

Comment:
```suggestion
        bonusNumber: Number(bonusNumber),
```
사소한거지만 airbnb 가이드에서는 `+` 를 통한 타입 변환보다 `Number` 을 권장하니 이렇게 바꾸면 좋을 것 같아요!

Reply:
좋은 정보 알려주셔서 감사합니다...! 👍 
1주차에 코드리뷰 하다가 보고 우와 하고 지금까지 쓰고 있는데 Number가 권장이군요..!